server.rb

```diff
@@ -5,33 +5,29 @@
 require_relative 'bot'
 require 'easy_translate'
 
-EasyTranslate.api_key= ENV['TRANSLATE_API_KEY']
-
-@@initial = 0
-@@users = {}
-
-def language(key)
-  if @@users[key] == "English"
-    {to: :en}
-  elsif @@users[key] == "Vietnamese"
-    {to: :vi}
-  elsif @@users[key] == "French"
-    {to: :fr}
-  else
-    {to: :en}
-  end
-end
-
 get '/callback' do
   if params["hub.mode"] == "subscribe" && params["hub.verify_token"] == "asdf123"
     params["hub.challenge"]
   else
     ap "Give me Parameters"
-    ap params
     "Unknown Parmeters"
   end
 end
 
+EasyTranslate.api_key= ENV['TRANSLATE_API_KEY']
+@@users = {}
+
+def language(id)
+  languages = EasyTranslate::LANGUAGES.invert
+  choice = @@users[id].downcase
+  languages.each do |key, value|
+    if key == choice
+      return {to: value.to_sym}
+    end
+  end
+  return nil
+end
```

Comment:
thanks for the feedback Harley,
I'll try to do it the way you suggested to see how it goes =). I did it the symbol way because I thought it needed a hash returned.

Concerning not returning `en`, Charles wanted me to give the user feedback if the language wasn't found.